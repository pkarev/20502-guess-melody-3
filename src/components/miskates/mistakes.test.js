import React from 'react';
import renderer from 'react-test-renderer';
import Mistakes from './mistakes.jsx';

it(`Render Mistakes`, () => {
  const tree = renderer
  .create(<Mistakes count={2}/>)
  .toJSON();

  expect(tree).toMatchSnapshot();
});
