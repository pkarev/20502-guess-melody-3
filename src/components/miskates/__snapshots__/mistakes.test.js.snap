// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Mistakes 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
</div>
`;
